config/fedora.yml

```diff
@@ -1,22 +1,20 @@
 development:
   user: fedoraAdmin
   password: fedoraAdmin
-  url: http://<%= ENV['FCREPO_HOST'] || 'fcrepo' %>:<%= ENV['FCREPO_PORT'] || 8080 %><%= ENV['FCREPO_PATH'] || '/fedora/rest' %>
-  base_path: <%= ENV['FCREPO_BASE_PATH'] || '/dev' %>
+  url: http://<%= ENV['FCREPO_HOST'] || 'localhost' %>:<%= ENV['FCREPO_DEVELOPMENT_PORT'] || ENV['FCREPO_PORT'] || 8984 %>/<%= ENV['FCREPO_REST_PATH'] || 'rest' %>
+  base_path: <%= ENV['FCREPO_DEV_BASE_PATH'] || ENV['FCREPO_BASE_PATH'] || '/dev' %>
```

Comment:
1. Nitpicky, but can we use either `DEV` or `DEVELOPMENT` instead of both? 
2. What should the values for these be? Should defaults be added to `.env`?

Reply:
Where are they listed?

Reply:
`url: http://<%= ENV['FCREPO_HOST'] || 'localhost' %>:<%= ENV['FCREPO_DEVELOPMENT_PORT'] || ENV['FCREPO_PORT'] || 8984 %>/<%= ENV['FCREPO_REST_PATH'] || 'rest' %>`

at the end, this example has a default of rest